wp-content/themes/goonj-crm/templates/form-check-user.php

```diff
@@ -10,23 +10,25 @@
 <div class="text-center">
 	<form class="logged-out wp-block-loginout" action="<?php echo home_url(); ?>" method="POST">
 		<?php if ($message) : ?>
-				<div class="message">
-					<?php echo $message; ?>
-				</div>
+			<div class="message">
+				<?php echo $message; ?>
+			</div>
 		<?php endif; ?>
 		<input type="hidden" name="action" value="goonj-check-user" />
-		<div class="d-grid">
-			<label for="email">Email</label>
-			<input type="email" id="email" name="email" required value="<?php echo isset($_POST['email']) ? htmlspecialchars($_POST['email']) : ''; ?>">
-		</div>
-		<br>
-		<div class="d-grid">
-			<label for="phone">Phone Number</label>
-			<input type="tel" id="phone" name="phone" required value="<?php echo isset($_POST['phone']) ? htmlspecialchars($_POST['phone']) : ''; ?>">
-		</div>
-		<br>
-		<p class="login-submit">
-			<input type="submit" class="button button-primary w-520" value="Submit">
-		</p>
+		<?php if (!$message) : ?>
+			<div class="d-grid">
+				<label class="font-sans" for="email">Email</label>
+				<input type="email" id="email" name="email" required value="<?php echo isset($_POST['email']) ? htmlspecialchars($_POST['email']) : ''; ?>">
```

Comment:
There is a more WordPress best practices way to refactor this code. Using WordPress functions and escaping methods ensures better security and consistency with WordPress standards. Here’s how you can do it:

1. Use esc_attr to escape the attribute value.
2. Use isset to check if the `$_POST` variable is set.
3. Use `sanitize_email` to sanitize the email input.

```suggestion
				<input type="email" id="email" name="email" required value="<?php echo isset($_POST['email']) ? esc_attr(sanitize_email($_POST['email'])) : ''; ?>">
```